plugins/dns_service/app/controllers/dns_service/zones_controller.rb

```diff
@@ -44,13 +44,16 @@ def create
     end
 
     def edit
-      @zone = services.dns_service.find_zone(params[:id])
     end
 
     def update
-      @zone = services.dns_service.find_zone(params[:id])
-      @zone.attributes=params[:zone]
-      if @zone.save
+      update_attributes = {
```

Comment:
Actually you don't need a mapping here. It is already done here: https://github.com/sapcc/elektra/blob/master/plugins/dns_service/app/models/dns_service/zone.rb#L12-L18